### ISSUE TITLE
Module resolution error when importing components in 1.4.1

### ISSUE THREAD

**Issue by jeancochrane** [react-bootstrap/react-bootstrap#5601]
## Describe the bug

After the 1.4.1 release about an hour ago, my tests started failing on CI with a module resolution error on every import:

```
Module not found: Error: Can't resolve 'react-bootstrap/Navbar'
```

The error gets raised for all components in my app. So far my workaround has been to pin to 1.4.0.

## To Reproduce

1. Run a standard module import of a component, e.g. `import Navbar from 'react-bootstrap/Navbar'
2. Attempt to build the project
3. Confirm that the project does not build

## Reproducible Example

I'm working on a reproducible example now, just wanted to get this in front of you in case you have thoughts about what might be going wrong!

## Expected behavior

Module imports should continue to work on 1.4.1.

## Environment (please complete the following information)

- Operating System: macOS Catalina
- React-Bootstrap Version 1.4.1

## Additional context

I tried to check if this update was intentional in the 1.4.1 release but I couldn't find anything. Apologies if this is user error!

**Comment by jeancochrane** [comment]
Dupe of https://github.com/react-bootstrap/react-bootstrap/issues/5600.

**Comment by kyletsang** [comment]
Sorry for the hiccups.  This should be fixed in 1.4.3
